abstract-sdk.d.ts

```diff
@@ -1855,8 +1855,10 @@ type CommandOptions = {
   apiUrl: string | Promise<string>,
   analyticsCallback: AnalyticsCallback,
   assetUrl: string | Promise<string>,
+  clientSecret?: string | Promise<string>,
   previewUrl: string | Promise<string>,
   shareId?: () => Promise<string | ShareDescriptor | ShareUrlDescriptor | void>,
+  redirectUri?: string | Promise<string>,
```

Comment:
```suggestion
  redirectUri?: string,
```

We're not supporting Promises here